### DISCUSSION TITLE
How to configure different namespace for producers and consumers.

### DISCUSSION THREAD

**Discussion by sualva** [majusko/pulsar-java-spring-boot-starter#205] [Q&A] [answered]
Hi Mario,
      
          Had a question. I am using this library in a spring boot application. I have publishers and consumers implemented in the same application, the publisher and consumers have different namespaces. How do I go about it? The only place I am seeing  where the namespace is configured is in the property "pulsar.namespace=",  and I see that both the publisher and consumer using the same UrlBuildService.buildTopicUrl(). Am I missing something?

       Appreciate your response in advance.

Regards
Sujana

**Comment by majusko** [comment, marked as answer]
Hey @sualva - the customisable namespaces are now part of the `1.1.1` release and you can now set the different namespaces for every consumer you use.
